Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
 RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
     go build -o bin/go-mysql-elasticsearch ./cmd/go-mysql-elasticsearch && \
-    cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch
-
-ENTRYPOINT ["go-mysql-elasticsearch"]
+    cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch && \
+    apk add --no-cache tini mariadb-client
 
+ENTRYPOINT ["/sbin/tini","--","go-mysql-elasticsearch","-config=/go/src/github.com/siddontang/go-mysql-elasticsearch/etc/river.toml"]
```

Comment:
How can the users use the customized config? maybe we should remove `-config` here.

Another way is use volumn to let the user bind the config.

Reply:
Maybe we could add a COPY or ADD command to copy the customized config to the container, like this:

```
COPY ./river.toml /go/src/github.com/siddontang/go-mysql-elasticsearch/etc/river.toml
```

Reply:
After a short thought, I think it is better to remove `-config`..